app.js

```
@@ -1,21 +1,27 @@
 var bodyParser = require('body-parser');
-var log4js = require("log4js");
-
+const log4js = require("log4js");
+const cookieParser = require('cookie-parser');
+const init_db = require('./model/init_db');
+const log4js = require("log4js");
+const products = require('./routes/products');
+const init_db = require('./model/init_db');
+const login = require('./routes/login');
+const products = require('./routes/products');
 var init_db = require('./model/init_db');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'init_db'. Consider to delete it or rename it.

'init_db' is defined here:


(lint/suspicious/noRedeclare)